Client/Frontend/Settings/Clearables.swift

```diff
@@ -126,24 +75,11 @@ class CacheClearable: Clearable {
             // Remove the basic cache.
             URLCache.shared.removeAllCachedResponses()
             
-            var err: Error?
-            for item in ["Caches", "Cookies", "WebKit"] {
-                do {
-                    try deleteLibraryFolderContents(item, validateClearedExceptFor: ["Snapshots"])
-                } catch {
-                    err = error
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes(), modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
+                ImageCache.shared.clear()
+                URLCache.shared.setupBraveDefaults()
+                result.fill(Maybe<()>(success: ()))
             }
-            if let err = err {
-                return result.fill(Maybe<()>(failure: ClearableErrorType(err: err)))
-            }
-            
-            // Clear image cache
-            ImageCache.shared.clear()
-            
-            // Leave the cache off in the error cases above
-            URLCache.shared.setupBraveDefaults()
```

Comment:
Looks like `setupBraveDefaults` and the file containing it can be also removed then.

But now I see one place with `URLCache.shared.memoryCapacity = 0`, but no other place when we set different values to it.
Should we still set memory/disk capacities somewhere after the new changes?

Reply:
The memory capacity is set to 0 to flush current cache and then reset to defaults. And since we are using `allWebsiteDataTypes` which covers all types of caches (memory/disk) we can safely remove this. .. Removed. Thanks, don't why I am missing these so much :P